pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1301,14 +1301,21 @@ func (ctrl *PersistentVolumeController) provisionClaimOperation(claimObj interfa
 		return
 	}
 
+	// Use StorageClass.ReclaimPolicy or fall back to Delete if it's somehow blank
+	reclaimPolicy := storageClass.ReclaimPolicy
+	if reclaimPolicy == "" {
```

Comment:
under what conditions do we think this would happen?  We don't (in general) want defaulting logic in multiple places.